### PR TITLE
Add API support for disabling count tool

### PR DESCRIPTION
**Pull request by darrenchann**
added support to disable count measurement tool on iOS/Android

`disabledTools={[Config.Tools.annotationCreateLine, Config.Tools.annotationCreateRectangle, Config.Tools.annotationCountTool]}`

image

### CODE REVIEW

File: android/src/main/java/com/pdftron/reactnative/utils/Constants.java [hunk 1]
```diff
@@ -89,6 +89,7 @@ public final class Constants {
     public static final String TOOL_BUTTON_EDIT = "editToolButton";
     public static final String TOOL_BUTTON_ADD_PAGE = "addPageButton";
     public static final String TOOL_BUTTON_INSERT_PAGE = "insertPageButton";
+    public static final String TOOL_BUTTON_COUNT = "countToolButton";
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/reactnative/utils/Constants.java:92]: don't think we need this one, customer can just use the tool version

File: ios/RNTPTDocumentView.h [hunk 2]
```diff
@@ -56,6 +56,7 @@ static NSString * const PTInsertFromImageButton = @"insertFromImageButton";
 static NSString * const PTInsertFromPhotoButton = @"insertFromPhotoButton";
 static NSString * const PTInsertFromDocumentButton = @"insertFromDocumentButton";
 static NSString * const PTInsertFromScannerButton = @"insertFromScannerButton";
+static NSString * const PTCountToolButton = @"countToolButton";
```
Comment (sgong-pdftron) [inline ios/RNTPTDocumentView.h:59]: same here

File: lib/src/Config/Config.js [hunk 3]
```diff
@@ -48,6 +48,7 @@ export const Config = {
         addPageButton: 'addPageButton',
         insertPageButton: 'insertPageButton',
         saveCroppedCopyButton: 'saveCroppedCopyButton',
+        countToolButton: 'countToolButton',
```
Comment (sgong-pdftron) [inline lib/src/Config/Config.js:51]: remove

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/pdftron/reactnative/utils/Constants.java (hunk 1 above)
Comment (darrenchann) [reply android/src/main/java/com/pdftron/reactnative/utils/Constants.java]: removed

**Review by sgong-pdftron — APPROVED** [review]
ok on android, please ensure fully tested and working

**Review by dluco — APPROVED** [review]

**Pull request merged**